src/main/java/seedu/module/model/task/Workload.java

```diff
@@ -0,0 +1,83 @@
+package seedu.module.model.task;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.module.commons.util.AppUtil.checkArgument;
+
+import java.util.InputMismatchException;
+
+/**
+ * Enum to set workload levels of tasks.
+ */
+public class Workload {
+    public static final String LOW_WORKLOAD = "Low";
+    public static final String MEDIUM_WORKLOAD = "Medium";
+    public static final String HIGH_WORKLOAD = "High";
```

Comment:
can we use enum? just a suggestion

Reply:
I was considering using enum but I couldn't get the constructors to work. Maybe I'll look into it in v1.3.